requests/backend/README.md

```
@@ -0,0 +1,51 @@
+const { data } = useQuery({
+  queryKey: ['dashboard'],
+  queryFn: () => getDashboard(accessToken),
+  enabled: !!accessToken,
+});
+```
+
+## Configuration
+
+Set `NEXT_PUBLIC_API_URL` environment variable:
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)